synapse/handlers/acme.py

```diff
@@ -71,7 +72,18 @@ def start_listening(self):
         # want it to control where we save the certificates, we have to reach in
         # and trigger the registration machinery ourselves.
         self._issuer._registered = False
-        yield self._issuer._ensure_registered()
+
+        try:
+            yield self._issuer._ensure_registered()
+        except Exception:
```

Comment:
hrmmm... swallowing all errors here and turning them into the same error message doesn't sound right. can we be more selective? or at least pass through the text of the original exception?

Does this error text actually appear in the console, or does it go to the logs, or what? `ConfigError` has special treatment to send it to the console when it is thrown in the early stages of synapse startup, but I don't know if that covers this codepath.